packages/math-input/src/types.ts

```diff
@@ -87,6 +87,14 @@ export type LayoutProps = {initialBounds: Bound};
 
 export type ClickKeyCallback = (key: Key) => void;
 
+export type KeypadPageType =
+    | "Geometry"
+    | "Operators"
+    | "Numbers"
+    | "Fractions"
+    | "Extras"
+    | "Dismiss";
```

Comment:
Ugh, now "Dismiss" feels wrong; though it's not really a "Tab" either. No action needed, just pointing it out.